tests/Unit/Core/ResponseTest.php

```
@@ -25,6 +25,8 @@ protected function setUp(): void
         $this->csrf = $this->createMock(CSRF::class);
         $this->csrf->method('getToken')->willReturn('test-csrf-token');
         $this->response = new Response($this->csrf);
+        // Reset static nonce so each test starts clean
+        (new \ReflectionProperty(Response::class, 'nonce'))->setValue(null, '');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '29', column '14'). (undefined)

(MissingImport)